addons/mail/static/src/core/web/chatter.js

```diff
@@ -360,16 +368,17 @@ export class Chatter extends Component {
 
     async scheduleActivity() {
         this.closeSearch();
-        const schedule = async (threadId) => {
-            await this.activityService.schedule(this.props.threadModel, [threadId]);
-            this.load(this.props.threadId, ["activities", "messages"]);
+        const schedule = async (threadModel, threadId) => {
+            const thread = this.threadService.getThread(threadModel, threadId);
```

Comment:
But `getThread` is not the method we want here.
Except inside `changeThread`, it should probably never be called.

There is this code inside id, that we don't want here:
```js
            // to force a reload
            thread.status = "new";
            ```